26-register_registration/creator_navigates_to_description_tab.feature

```diff
@@ -48,3 +49,13 @@ Feature: Creator navigates to Description tab
     Given Creator has added a Project
     When they click the Remove Project icon
     Then they see the Project is removed from the list of selected Projects
+
+  @2446
+  Scenario: Creator adds a Vocabulary
+    Given Creator begins registering a Registration in the Wizard
+    And they navigate to the Description tab
+    And their institution has a Vocabulary set as "Allowed"
+    And this Vocabulary is not set as "Default"
```

Comment:
Should there be / Is there a scenario for the case where the Vocabulary *is* set to default?